Sources/Networking/Operations/Handling/CustomerInfoResponseHandler.swift

```diff
@@ -59,3 +112,52 @@ extension CustomerInfoResponseHandler {
     }
 
 }
+
+private extension CustomerInfoResponseHandler {
+
```

Comment:
having this whole feature at the response handling level honestly feels a bit off to me - none of the other handlers do anything this sophisticated, and this one already takes care of subscriber attributes errors, it feels like it might be handling a bit too much stuff. 
I'd also love to see all logic relevant to Offline Entitlements live in the OfflineEntitlements group, so that if a maintainer wants to see how that feature works, they can find everything they need in a single place

Reply:
I moved 99% of this code out of here and into `OfflineEntitlements/CustomerInfo+OfflineEntitlements.swift`

As for whether this will be called from here or the logic will be used elsewhere, I'm looking at all the edge cases and how to handle them (and how Android did it) to decide that.